aggregator/README.md

```diff
@@ -8,21 +8,39 @@ To build in docker, run docker build from the root of the project
 docker build -f aggregator/Dockerfile -t concepts-aggregator .
 ```
 
-You can then run it locally to see the lists of Concepts that would be returned by 
+Or (better), docker compose build from here.
+
+```shell
+  docker compose build
+```
+
+You can then run it locally. First start elasticsearch:
+```shell
+ docker compose run -d --service-ports elasticsearch 
+```
+
+Then, to see the lists of Concepts that would be returned by 
 a list of Work ids, thus:
 
 ```shell
-docker run concepts-aggregator uk4kymkq yn8nshmc  
+docker compose run aggregator uk4kymkq yn8nshmc  
```

Comment:
interesting to consider the commercial significance of the disappearance of the `-` in docker-compose